### DISCUSSION TITLE
C4FM P25 v1 Manual Add Decoder

### DISCUSSION THREAD

**Discussion by linuxnut79** [DSheirer/sdrtrunk#1940] [Ideas]
I have had the idea of listing to each Frequency my area p25 system has.  Our system has 5 frequencies, 2 are control and 3 are talk.  850 range.  i have a lot of SDR dongles and i can use them to listen to each talk freq.  However SDRTRUCK dosent have the option to add a C4FM decoder.   Could this be an added feature so i can just ignore the control and just listen to the talk freq?

**Comment by cfc62** [comment]
You can already do this in SDR Trunk.

**Reply by linuxnut79** [reply]
How many I ask?  What am I missing because I can't find a way to.
